### DISCUSSION TITLE
bluetooth devices appear grayed out on the recent connections list

### DISCUSSION THREAD

**Discussion by yodavid1984** [blueman-project/blueman#2343] [Q&A]
not all of them, actually, but the one i want to connect now is one of them. i tried removing the software and reinstalling. unplugging and replugging my dongle, charging my device (nintendo wii u pro controller)

i'm on Pop!_os 22.04

**Comment by cschramm** [comment]
Do they have a tooltip, say `Adapter for this connection is not available`? While the tooltip might not necessarily show up, that should be the only reason for the items to be insensitive.

**Reply by yodavid1984** [reply]
https://ibb.co/cQXg2CV

this is how it appears to me. by the way i just tested it on a notebook that runs the same OS and it worked just fine

the computer where i'm having this problem is a desktop with a bluetooth dongle. it did work in the past, nevertheless

**Reply by cschramm** [reply]
I cannot see if it shows a tooltip or not. Anyway, I'm rather sure that those devices had been connected with an adapter that's logically not available. Pairing them with the available adapter should work fine.

**Reply by yodavid1984** [reply]
when i go to devices and click to find new devices, it doesn't find the controller. only the device that is not grayed out in that print

now i just tested, and it can detect also my cell phone when i activate its bluetooth function

**Reply by yodavid1984** [reply]
do you have an idea what can I try?

**Reply by cschramm** [reply]
It clearly sounds like the controller is simply not discoverable. The Internets tell me that it has a sync button that puts it into pairing mode.

**Reply by yodavid1984** [reply]
i tried pairing it in a notebook that i have here that runs the exact same OS and it works fine.

**Reply by cschramm** [reply]
Cleaning some cached state from `/var/lib/bluetooth/` might help then.

**Reply by yodavid1984** [reply]
there are two folders there. should i just delete them?

**Reply by cschramm** [reply]
To reset all bluetoothd state, you'd clean that directory, yes, ideally while bluetoothd is not running.
